CHANGELOG.md

```diff
@@ -15,10 +15,14 @@ All notable changes to this project will be documented in this file.
 - CKKS: added the `Evaluator.Average` method.
 - CKKS: added `DomainSwitcher` type for conversion between Standard and Conjugate Invariant variants of CKKS.
 - CKKS: `Encoder.Encode*` methods now support both  `[]complex128` and `[]float64` as input.
+- CKKS: uniformized order of inputs of the `Encoder` interface.
```

Comment:
"Input" is ambiguous: "uniformized the arguments' position for all methods of the `Encoder` interface"?

Reply:
Also, to be more uniform with the rest of the library, `plaintext` should be the last argument (it receives the output).